iai_donbot_description/robots/iai_donbot_with_refills_finger.xacro

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
```

Comment:
This file should probably be called `iai_donbot_with_refills_finger.urdf.xacro` as well.

Reply:
ok I changed it